screens/auth/ChangePasswordScreen.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {Text, StyleSheet, View, Alert} from 'react-native';
+import {Auth} from 'aws-amplify';
+import PropTypes from 'prop-types'
+import ChangePasswordForm from "../../components/auth/ChangePasswordForm";
+
+
+export default class ChangePasswordScreen extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            email: '',
+            oldPassword: '',
+            newPassword: '',
+            confirmNewPassword: '',
+        };
+        this.handleFormChange = this.handleFormChange.bind(this);
+    }
+
+    handleFormChange(field, value) {
+        this.setState({[field]: value});
+    }
+
+    handlePasswordChange = () => {
+        const {oldPassword, newPassword, confirmNewPassword} = this.state;
+        const email = this.state.email;
```

Comment:
Is there a reason why you're not combining this line with the line above it?

Reply:
woops, nope i'll merge them